### PR TITLE
Add community resources

### PR DESCRIPTION
**Pull request by petermbenjamin**
/sig contributor-experience
/assign @castrojo

### CODE REVIEW

**Comment by petermbenjamin** [comment]
This is a pull request per a conversation with @castrojo [here](https://discuss.kubernetes.io/t/how-do-you-keep-up-with-kubernetes/3190/6)

File: communication/resources/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+- The resource must be free.
+
+## Resources
+
+### Audio
+
+- [PodCTL](https://twitter.com/PodCTL)
+- [Kubernetes Podcast](https://kubernetespodcast.com)
+- [The New Stack Podcasts](https://thenewstack.io/podcasts/)
+- [SoundCloud](https://soundcloud.com/search?q=kubernetes)
```
Comment (jdumars) [inline communication/resources/README.md]: I'm not a fan of an undifferentiated search link. Granted, the hits will most likely be appropriate, but there is no way to ensure that. I'd either strike this, or add the most relevant one(s).

File: communication/resources/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- [KubeWeekly](https://twitter.com/kubeweekly)
+- [Kubedex](https://kubedex.com/category/blog/)
+- [Kubernetes Blog](https://kubernetes.io/blog/)
+- [Kubernetes Changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md)
+- [Kubernetes Enhancements Repo](https://github.com/kubernetes/enhancements)
+- [Kubernetes Forum](https://discuss.kubernetes.io)
+- [Last Week in Kubernetes Development](http://lwkd.info)
+- [Medium](https://medium.com/tag/kubernetes)
+- [Reddit](https://www.reddit.com/r/kubernetes)
+- [The New Stack Books](https://thenewstack.io/ebooks/)
```
Comment (jdumars) [inline communication/resources/README.md]: This doesn't feel in alignment with other resources.

File: communication/resources/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+### Text
+
+- [CNCF Blog](https://www.cncf.io/newsroom/blog/)
+- [Dev.To](https://dev.to/t/kubernetes)
+- [Heptio Blog](https://blog.heptio.com)
+- [KubeTips](http://kubetips.com)
+- [KubeWeekly](https://twitter.com/kubeweekly)
+- [Kubedex](https://kubedex.com/category/blog/)
+- [Kubernetes Blog](https://kubernetes.io/blog/)
+- [Kubernetes Changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md)
```
Comment (jdumars) [inline communication/resources/README.md]: I would omit. Inner-project resources should probably be referenced elsewhere to keep the spirit of this list.

File: communication/resources/README.md (hunk 1 above)
Comment (petermbenjamin) [reply communication/resources/README.md]: Would this be better?
```suggestion
- [SoundCloud](https://soundcloud.com/search/sounds?q=kubernetes&filter.genre_or_tag=technology)
```

If not, I don't mind striking it.

File: communication/resources/README.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+- [Kubernetes Enhancements Repo](https://github.com/kubernetes/enhancements)
+- [Kubernetes Forum](https://discuss.kubernetes.io)
+- [Last Week in Kubernetes Development](http://lwkd.info)
+- [Medium](https://medium.com/tag/kubernetes)
+- [Reddit](https://www.reddit.com/r/kubernetes)
+- [The New Stack Books](https://thenewstack.io/ebooks/)
+- [Weaveworks Blog](https://www.weave.works/blog/category/kubernetes/)
```
Comment (jdumars) [inline communication/resources/README.md]: How do we feel about vendors on this list? I have nothing against Weaveworks, or Heptio (quite the opposite), but they are the only vendors in here. I'd either set the precedent fully, and add all of the other great vendor content out there, or keep it completely vendor-agnostic. My vote is the latter.

File: communication/resources/README.md (hunk 2 above)
Comment (petermbenjamin) [reply communication/resources/README.md]: Would it be better if split up into the specific Kubernetes ebooks?
- https://thenewstack.io/ebooks/kubernetes/kubernetes-deployment-and-security-patterns/
- https://thenewstack.io/ebooks/kubernetes/kubernetes-solutions-directory/
- https://thenewstack.io/ebooks/kubernetes/ci-cd-with-kubernetes/
- https://thenewstack.io/ebooks/kubernetes/state-of-kubernetes-ecosystem/
- https://thenewstack.io/ebooks/use-cases/use-cases-for-kubernetes/

File: communication/resources/README.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+### Video
+
+- [BrightTALK Webinars](https://www.brighttalk.com/search/?q=kubernetes)
+- [Ceph YouTube Channel](https://www.youtube.com/channel/UCno-Fry25FJ7B4RycCxOtfw)
+- [CNCF YouTube Channel](https://www.youtube.com/channel/UCvqbFHwN-nwalWPjPUKpvTA)
+- [CoreOS YouTube Channel](https://www.youtube.com/user/coreoslinux)
```
Comment (jdumars) [inline communication/resources/README.md]: Chances are the content here will become increasingly out of date. I would either qualify as "historical info" or omit.

File: communication/resources/README.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+- [Kubernetes YouTube Channel](https://www.youtube.com/channel/UCZ2bu0qutTOM0tHYa_jkIwg)
+- [Lachlan Evenson YouTube Channel](https://www.youtube.com/channel/UCC5NsnXM2lE6kKfJKdQgsRQ)
+- [Rancher YouTube Channel](https://www.youtube.com/channel/UCh5Xtp82q8wjijP8npkVTBA)
+- [Rook YouTube Channel](https://www.youtube.com/channel/UCa7kFUSGO4NNSJV8MJVlJAA)
+- [Tigera YouTube Channel](https://www.youtube.com/channel/UC8uN3yhpeBeerGNwDiQbcgw)
+- [Udacity Course](https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615)
+- [Udemy Courses](https://www.udemy.com/courses/search/?courseLabel=&sort=relevance&q=kubernetes&price=price-free)
+- [Weaveworks YouTube Channel](https://www.youtube.com/channel/UCmIz9ew1lA3-XDy5FqY-mrA/featured)
+
+### Miscellaneous
```
Comment (jdumars) [inline communication/resources/README.md]: Consider omitting the in-community links, and make these Learning Resources instead.

File: communication/resources/README.md (hunk 4 above)
Comment (petermbenjamin) [reply communication/resources/README.md]: I am of the former opinion. 
Lots of vendors are contributing great resources to the community in all forms.
As for adding all the other great vendor content out there, I welcome contributions from anyone, as long as they are not undifferentiated search links.

File: communication/resources/README.md (hunk 1 above)
Comment (petermbenjamin) [reply communication/resources/README.md]: No problem. Will strike.

File: communication/resources/README.md (hunk 3 above)
Comment (petermbenjamin) [reply communication/resources/README.md]: Will omit.

File: communication/resources/README.md (hunk 5 above)
Comment (petermbenjamin) [reply communication/resources/README.md]: Removed.

File: communication/resources/README.md (hunk 6 above)
Comment (petermbenjamin) [reply communication/resources/README.md]: Removed in-community links. Renamed section to "Learning Resources". Moved learning resources into this section.

**Comment by jdumars** [comment]
/lgtm
nice work!

**Pull request merged**